docs/src/pages/[platform]/getting-started/usage/nextjs/react.mdx

```diff
@@ -118,3 +118,73 @@ Then, replace the contents of `pages/index.js` with the code below, restart the
   </TabItem>
 </Tabs>
 
+### Connected Components
```

Comment:
Rather than putting this callout at the bottom, should we put it at the top before the tutorial?

Reply:
We should add an info Alert on the Authenticator page that links to this section